OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -253,7 +289,7 @@ private async void InitializeLibVLC(InitializedEventArgs eventArgs)
             // Create LibVLC instance.
             LibVLC = new LibVLC(eventArgs.SwapChainOptions);
             MediaPlayer = new MediaPlayer(LibVLC);
-
+            MediaTrackService.Initialize(ref mediaPlayer);
```

Comment:
```suggestion
            MediaTrackService.Initialize(ref MediaPlayer);
```

Reply:
I prefer to use the private fields when inside its own class.

Reply:
Wont work due to ref.

Reply:
it seems we already use this property/field inconsistently and that we can't use the property here.